### PR TITLE
TOMEE-2602 - add translate in PT to examples/tomee-jms-portability

### PR DESCRIPTION
**Pull request by robinsonvs**
ticket of Jira = https://issues.apache.org/jira/browse/TOMEE-2602

### CODE REVIEW

File: examples/tomee-jms-portability/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+(Arquivo não incluido neste repositório git)
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<tomee>
+</tomee>
+----
+
+
+### catalina.properties
```
Comment (Daniel-Dos) [inline examples/tomee-jms-portability/README_pt.adoc]: change "convinentemente" to "convenientemente"

File: examples/tomee-jms-portability/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Este arquivo pode ser usado para fornecer convinentemente argumentos java -D como uma entrada para tomee. No entanto, no mundo real, esses parâmetros seriam configurados usando variáveis de ambiente para evitar a adição de propriedades específicas do aplicativo no tomee conf (Arquivo não incluído neste repositório git).
```
Comment (Daniel-Dos) [inline examples/tomee-jms-portability/README_pt.adoc]: change "convinentemente" to "convinentemente"

Comment (robinsonvs) [reply examples/tomee-jms-portability/README_pt.adoc]: for me both are the same

Comment (robinsonvs) [reply examples/tomee-jms-portability/README_pt.adoc]: "convenientemente"

Comment (robinsonvs) [reply examples/tomee-jms-portability/README_pt.adoc]: Done, thanks @Daniel-Dos

**Comment by cesarhernandezgt** [comment]
Thanks, @robinsonvs  for the contribution and @Daniel-Dos for the review!

**Pull request merged**